pom.xml

```
@@ -27,11 +27,11 @@
   </modules>
 
   <properties>
-    <version.camunda>7.15.0</version.camunda>
+    <version.camunda>7.16.0-alpha5</version.camunda>
```

Reminder to change that back to 7.16.0
```suggestion
    <version.camunda>7.16.0</version.camunda>
```